README.md

```diff
@@ -6,6 +6,18 @@ Spito has been designed to allow users easily swap
 and edit configuration files. It's user-friendly and
 lays the foundations for auto-repair tools.
 
+## Installation
+Only for local user
+```bash
+go install
```

Comment:
I wonder about adding info that `~/go/bin` should be exported in `.bashrc`, `.zshrc` or anything like this

Reply:
I happens by default